src/main/java/com/github/onsdigital/babbage/response/util/CacheControlHelper.java

```diff
@@ -8,10 +8,12 @@
 import javax.servlet.http.HttpServletResponse;
 
 import static com.github.onsdigital.logging.v2.event.SimpleEvent.info;
+import static com.github.onsdigital.babbage.configuration.ApplicationConfiguration.appConfig;
 
 /**
  */
 public class CacheControlHelper {
+    private static final boolean legacyCacheAPIEnabled = appConfig().babbage().isLegacyCacheAPIEnabled();
```

Comment:
Small one, but in Java it's customary to name constants in all capital letters and underscores.

Reply:
I noticed that other constants were declared in this way within Babbage so I thought I would keep it consistent, but can make the update if we think it's better!

Reply:
Yes, unfortunately there is a mixture of styles. Let's leave it like this then, I haven't seen any mentions of a preferred style, so if there is one, someone at ONS will let us know during the review.